config/settings.yml

```diff
@@ -5,6 +5,8 @@ chef:
 doorkeeper:
   administrators:
     - rainbowdash
+email_from_address: 'Chef Notifications <notifications@chef.io>'
```

Comment:
This particular line is for testing, but the cookbook for oc-id cookbook writes out this same file for production use, so there will be a line like this in this particular file in production as well.

For the sake of this particular task, I'd like to leave the IP addresses alone.  We can open separate tickets to correct the IPs, because yeah, everything uses that 33.x.x.x.